dashboard/test/models/pd/enrollment_test.rb

```diff
@@ -18,6 +18,94 @@ class Pd::EnrollmentTest < ActiveSupport::TestCase
     assert_equal enrollment, found_enrollment
   end
 
+  # charter + zip  --  success
+  test 'district by type charter and zip, validation succeeds' do
+    enrollment = build :pd_enrollment, school_type: "charter", school_zip: 98144, school_state: nil, school_district_id: nil
+    assert enrollment.valid?
+    assert_equal 0, enrollment.errors.messages.count
```

Comment:
If I'm not mistaken, these are unnecessary.

Per http://guides.rubyonrails.org/active_record_validations.html:
```
After Active Record has performed validations, any errors found can be accessed through the errors.messages instance method, which returns a collection of errors. By definition, an object is valid if this collection is empty after running validations.
```